fulp_modules/Pet_Objectives/objective_items.dm

```diff
@@ -0,0 +1,109 @@
+/datum/objective_item/steal/iandog
+	name = "Ian, the Head of Personnel's pet corgi, alive."
+	targetitem = /obj/item/pet_carrier
+	difficulty = 20
+	excludefromjob = list("Head of Personnel")
+	altitems = list(/obj/item/clothing/head/mob_holder)
+
+/datum/objective_item/steal/iandog/New()
+	special_equipment += /obj/item/lazarus_injector
+	..()
+
+/datum/objective_item/steal/iandog/check_special_completion(obj/item/I)
+	if(istype(I, /obj/item/pet_carrier))
+		var/obj/item/pet_carrier/C = I
+		for(var/mob/living/simple_animal/pet/dog/corgi/ian/D in C)
+			if(D.stat != DEAD)//checks if pet is alive.
+				return TRUE
+		for(var/mob/living/simple_animal/pet/dog/corgi/puppy/D in C)
+			if(D.stat != DEAD)//checks if pet is alive.
+				if(D.desc == "It's the HoP's beloved corgi puppy.")
+					return TRUE
+	if(istype(I, /obj/item/clothing/head/mob_holder))
+		var/obj/item/clothing/head/mob_holder/C = I
+		for(var/mob/living/simple_animal/pet/dog/corgi/ian/D in C)
+			if(D.stat != DEAD)//checks if pet is alive.
+				return TRUE
+		for(var/mob/living/simple_animal/pet/dog/corgi/puppy/D in C)
+			if(D.stat != DEAD)//checks if pet is alive.
+				if(D.desc == "It's the HoP's beloved corgi puppy.")
+					return TRUE
+	return FALSE 
+
+/datum/objective_item/steal/poly
+	name = "Poly, the Chief Engineer's pet parrot, alive"
+	targetitem = /obj/item/pet_carrier
+	difficulty = 30
+	excludefromjob = list("Chief Engineer")
+	altitems = list(/obj/item/clothing/head/mob_holder)
+
+/datum/objective_item/steal/poly/New()
+	special_equipment += /obj/item/lazarus_injector
```

Comment:
What if they also got a special pet carrier that fit in their bag? Poly is the only pet theft objective that cant be picked up, and pet carriers dont fit in anything

Reply:
That is a great idea though I'm not sure exactly how to make that.